wire/core/ModulesInstaller.php

```diff
@@ -266,14 +266,14 @@ public function isUninstallable($class, $returnReason = false) {
 				$reason = $this->_("Module is permanent");
 			} else {
 				$dependents = $this->getRequiresForUninstall($class);
-				if(count($dependents)) $reason = $this->_("Module is required by other modules that must be removed first");
+				if(count($dependents)) $reason = $this->_("Module is required by other modules that must be removed first: "  . implode(', ', $dependents));
```

Comment:
Note: this is a translatable string. For use cases like this you’ll want to use sprintf() or add the list of values after the language method call.